README.md

```diff
@@ -66,3 +66,18 @@
 #### Notes
```

Comment:
Is the Notes the appropriate place for the instructions?

Reply:
moved